models/mart/fct_daily_electricity_usage_carbon_emission.sql

```diff
@@ -18,10 +18,36 @@ weighted_daily_intensity AS (
 SELECT
```

Comment:
I think we should rename this model to `fct_estimated_electricity_usage_carbon_emission` so the naming convention is aligned with the invoice one